lib/components/Voting.tsx

```diff
@@ -42,10 +64,35 @@ export default function Voting({
       </div>
     );
 
+  const currentStatementNumber = Math.min(
+    currentStatementIx + 1,
```

Comment:
Is it possible for `currentStatementIx >= statements.length`?

Reply:
Seems to be.

I initially tried using `{currentStatementIx} of {statements.length}`, but when voting finished, the counter displayed '3 of 2'. 

If you'd like, I can adjust the index logic instead of this workaround.

Reply:
Nope, this looks good